portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +25,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments_ = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<String>();
-    comments.add("This page looks great");
-    comments.add("nice job");
-    comments.add("perfect");
     String json = "{";
-    json += "\"commentOne\": ";
-    json += "\"" + comments.get(0) + "\"";
-    json += ", ";
-    json += "\"commentTwo\": ";
-    json += "\"" + comments.get(1) + "\"";      
-    json += ", ";
-    json += "\"commentThree\": ";
-    json += "\"" + comments.get(2) + "\""; 
-    json += "}";
+    for(int x=0; x<comments_.size(); x++){
```

Comment:
I added a comment before about this, a little spacing here would be nicer to read, so:
    for (int x=0; x<comments_.size(); x++) {

It breaks up the conditions and the if/following-statement.